### PR TITLE
Remove not used messages and add comments for used but marked as not by IDE

### PR DESCRIPTION
**Pull request by Beacze**

### CODE REVIEW

File: Kitodo/src/main/resources/messages/messages_de.properties [hunk 1]
```diff
@@ -93,146 +71,112 @@ MetsEditorMissingFile=Die Datei {0} kann im Ordner {1} nicht gefunden werden.
...
...te (Images); Import- und Exportschnittstellen f\u00FCr Metadaten und f\u00FCr Fremd-Digitalisate (zum Beispiel aus Russland); Controlling-Mechanismen\: welcher Partner ist wie weit mit seinen Arbeiten, welche Zeitschrift ist in welchem Bearbeitungsstadium etc.; Fehlermeldungen, Abschlie\u00DFen eines Arbeitsschrittes und Weiterleiten zur n\u00E4chsten Arbeitsstufe (eventuell hier auch Wechsel/\u00DCbergabe von einem Partner zum anderen Partner). Komponenten f\u00FCr das verteilte Workflowmanagement sind integriert worden, um die Verwaltung einer verteilten Produktion und Kommunikation zwischen den unterschiedlichen Partnern zu gew\u00E4hrleisten. F\u00FCr jeden einzelnen Workflowschritt kann eingestellt werden, ob er parallel zum vorherigen durchgef\u00FChrt werden kann oder erst nach dessen erfolgreichem Abschluss. Ferner werden Korrekturschleifen abgebildet, um in sp\u00E4teren Schritten festgestellte fehlerhafte Daten fr\u00FCherer Schritte zu vermerken. Die Applikation ist transparent konzipiert, somit ist es f\u00FCr jeden Projektpartner jederzeit m\u00F6glich, den Status der Teilschritte des Workflows inklusive Personenzuordnung f\u00FCr einzelne Werke detailliert einzusehen.
```
Comment (henning-gerhardt) [inline Kitodo/src/main/resources/messages/messages_de.properties]: This is used in `kitodo_config.properties` file as one of two standard possible values.

File: Kitodo/src/main/resources/messages/messages_de.properties [hunk 2]
```diff
@@ -1066,35 +844,31 @@ typeSetNewspaper=Zeitung
...
+# userSaving is used in userEdit.xhtml - line 39
 userSaving=Benutzer wird gespeichert...
 validieren=Validieren
 value=Wert
-valueIsRequired=Der Wert wird ben\u00F6tigt.
-verlag=Verlag
 verlinkungDesBildesMitStrukturelement=Verlinkung des Bildes mit dem gew\u00E4hlten Strukturelement
 verschiebenAbbrechen=Verschieben abbrechen
-verwendeter\ Katalog\:=Verwendeter Katalog\:
+verwendeter\ catalogue\:=Verwendeter Katalog\:
```
Comment (henning-gerhardt) [inline Kitodo/src/main/resources/messages/messages_de.properties]: Where is this value "verwendeter Katalog" or "verwendeter catalgue" are used?

File: Kitodo/src/main/resources/messages/messages_en.properties [hunk 3]
```diff
@@ -94,148 +72,112 @@ MetsEditorMissingFile=The file {0} can not be found in {1} folder.
...
 aktuellerOrdner=Current folder
-aktuellesPasswortFalsch=The current password is wrong
 allActiveUsers=all active users
 alleAuswaehlen=Select all
 alleSeiten=All pages
 alleTasksAusListeEntfernen=Remove all tasks
 alleTrefferDieserSeiteInMeinHomeverzeichnis=Put all hits for this page in my home directory
-allgemeineEigenschaften=General properties
 allgemeines=General
-allgemeinesText=Kitodo controls the organisation of the digitisation workflow. The software package for this cooperative and collaborative work environment is the central module
```
Comment (henning-gerhardt) [inline Kitodo/src/main/resources/messages/messages_en.properties]: See my comment in `messages_de.properties` to this line.

File: Kitodo/src/main/resources/messages/messages_en.properties [hunk 4]
```diff
@@ -1073,34 +844,29 @@ typeSetNewspaper=newspaper
...
 users=Users
+# userSaving is used in userEdit.xhtml - line 39
 userSaving=Saving user...
 validieren=Validate
-valueIsRequired=The value is required.
-verlag=Publishing house
 verlinkungDesBildesMitStrukturelement=Link the image with the selected structure element
 verschiebenAbbrechen=Cancel move
-verwendeter\ Katalog\:=Used Catalogue\:
 verwendeter\ catalogue\:=used catalogue\:
```
Comment (henning-gerhardt) [inline Kitodo/src/main/resources/messages/messages_en.properties]: Where is this entry used?

**Comment by henning-gerhardt** [comment]
There are merge conflicts  in `Kitodo/src/main/resources/messages/messages_de.properties` and `Kitodo/src/main/resources/messages/messages_en.properties`. Please solve them.

**Comment by Beacze** [comment]
Rebased

**Comment by henning-gerhardt** [comment]
There are merge conflicts in `Kitodo/src/main/resources/messages/messages_de.properties` and `Kitodo/src/main/resources/messages/messages_en.properties`. Please solve them.

**Pull request merged**